dracut/30afterburn/module-setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+check() {
+    return 0
+}
+
+depends() {
+    echo systemd network
+}
+
+install() {
+    inst_multiple afterburn
+
+    inst_simple "$moddir/azure-hostname.service" \
+        "$systemdutildir/system/azure-hostname.service"
+
+    mkdir -p "$initdir/$systemdsystemunitdir/initrd.target.requires"
+
+    ln -s "../azure-hostname.service" "$initdir/$systemdsystemunitdir/initrd.target.requires/azure-hostname.service"
```

Comment:
Maybe just `afterburn-hostname.service` ? Theoretically, other clouds could need this too in the future.

Reply:
Good point